frontend/src/metabase/dashboard/components/QuestionPicker/QuestionPicker.tsx

```diff
@@ -84,24 +84,24 @@ function QuestionPickerInner({
         onChange={handleSearchTextChange}
       />
 
-      {(hasDataAccess || hasNativeWrite) && !isEmbeddingSdk() && (
+      {(hasDataAccess || hasNativeWrite) && (
         <Flex gap="sm" mb="md" data-testid="new-button-bar">
           {hasDataAccess && (
             <Button
               variant="outline"
               className={S.newButton}
-              leftSection={<Icon name="insight" />}
-              onClick={() => onNewQuestion("notebook")}
+              leftSection={<Icon aria-hidden name="insight" />}
+              onClick={onNewQuestion}
             >
               {t`New Question`}
             </Button>
           )}
-          {hasNativeWrite && (
+          {hasNativeWrite && !isEmbeddingSdk() && (
```

Comment:
Condition moved from line 87. And we don't have a way to render SQL query builder on the SDK, so we don't need to display this button.